Philippines/bagongpanimula.md

```diff
@@ -0,0 +1,3 @@
+Dito nag-uumpisa ang iyong daan tungo sa pagbabago.
+
+Gusto mo bang [magpahinga](Philippines/tulog.md) or [mag-aral](Philippines/aral.md) pa? 
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.